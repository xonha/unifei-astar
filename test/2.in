5
00000
F0FF0
00000
F00TF
00000
