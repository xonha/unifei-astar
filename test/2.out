Tomou -170 de dano! (quanto mais negativo melhor)
